Google.PowerShell/Common/MeasurementProtocolService.cs

```diff
@@ -8,7 +8,9 @@
 using System.Linq;
 using System.Net;
 using System.Text;
+#if !CORECLR
```

Comment:
Please add a comment to call this out. What is the difference between the CORECLR build and the non-Core one? Is the feature using a different set of libraries, or removed entirely?